### PR TITLE
Fix/test hj

### PR DESCRIPTION
**Pull request by sueelly**
## #️⃣ Related Issue

Linked Feature backlog: 

## 📝 Purpose of PR

Please briefly explain your work in this PR.

## Contents

List of work items performed.

## Checklist

-   [ ] Does commit messages follow the convention?
-   [ ] Are variable names brief but descriptive?
-   [ ] Is the code clean and easy to understand?
-   [ ] If a new feature has been added, or a bug fixed, has a test been added to confirm good behavior?
-   [ ] Does the test successfully test edge and corner cases?

## 💬 Review Requirements (Optional)

Is there anything you can praise about this PR? Start with praise.
Make a review, leaving kind comments and suggesting changes where needed (to resolve the above).

### CODE REVIEW

File: src/test/java/ac/dnd/dodal/acceptance/goal/AchieveGoalAcceptanceTest.java [hunk 1]
```diff
@@ -18,6 +18,8 @@
 
 public class AchieveGoalAcceptanceTest extends AcceptanceTest {
 
+    Long goalId = 4L;
```
Comment (5hseok) [inline src/test/java/ac/dnd/dodal/acceptance/goal/AchieveGoalAcceptanceTest.java:21]: 다른 곳들과 달리 여기서는 private을 명시하지 않은 이유가 뭘까요?

File: src/test/java/ac/dnd/dodal/acceptance/goal/DeleteGoalAcceptanceTets.java
Comment (5hseok) [inline src/test/java/ac/dnd/dodal/acceptance/goal/DeleteGoalAcceptanceTets.java:1]: 여기 클래스명이 **Tets**로 잘못 작성되었네요 
**Test**로 수정해주세요!

Comment (5hseok) [inline src/test/java/ac/dnd/dodal/acceptance/goal/DeleteGoalAcceptanceTets.java:1]: 여기 클래스명이 **Tets**로 잘못 작성되었네요 
**Test**로 수정해주세요!

**Review by 5hseok — CHANGES_REQUESTED** [review]

File: src/test/java/ac/dnd/dodal/acceptance/goal/AchieveGoalAcceptanceTest.java (hunk 1 above)
Comment (sueelly) [reply src/test/java/ac/dnd/dodal/acceptance/goal/AchieveGoalAcceptanceTest.java:21]: private으로 명시하지 않은 것은 이유가 없어요. 컨벤션 맞춰 수정하겠습니다~

acceptanceTest를 묶는 이유가 매번 springboot context가 재수행되지 않고 하나를 재사용하기 위해서에요. 근데 이 경우 문제가 이전의 결과들이 누적이 됩니다. 여기서 다른 데에서도 사용하는 goal을 achieve해버리면 다른 테스트들에서 achieved된 goal이라며 예외가 발생하는 케이스들이 발생합니다
매 테스트 수행 후 db를 날리는 것도 생각했지만, 조회 로직 때문이라도 샘플 데이터는 필요하고, 그러면 매번 db를 초기화했다가 모든 데이터를 새로 넣어야하는데 그럴 필요까지 있나하는 생각이 들었습니다.
인수 테스트는 결국 사용자 시나리오를 테스트하는 거고, 그러면 여러 테스트들이 독립적이 아니라 하나의 플로우처럼 이루어질 수 있다고도 생각해서요
